core/controller/src/main/scala/org/apache/openwhisk/core/controller/ApiUtils.scala

```diff
@@ -341,7 +342,7 @@ trait WriteOps extends Directives {
     }) {
       case Success(entity) =>
         logging.debug(this, s"[PUT] entity success")
-        postProcess map { _(entity) } getOrElse complete(OK, entity)
+        postProcess map { _(entity) } getOrElse complete(OK)
```

Comment:
As @bdoyle0182 said, it is a breaking change, it would great to provide an option to operators.
So operators can freely choose whether to enable/disable the feature with a flag.
And it should be disabled by default.

You may find this useful.
https://github.com/apache/openwhisk/pull/4334/files

Reply:
Hi @style95, Thank you for the review. I have added the feature flag for this change. 
So, now I should revert the test files to the previous behaviour, is that correct?

Reply:
I think you can add test cases for both cases with the config enabled/disabled.

Reply:
Hi @style95 I see that for now the [tests have been written with Feature enabled.](https://github.com/apache/openwhisk/blob/master/tests/src/test/resources/application.conf.j2#L20)
Can you please point me to an example of the tests with feature flag enabled/disabled?